modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/dto/v1_0/util/ObjectStateFlowUtil.java

```diff
@@ -111,6 +107,14 @@ private static ObjectStateTransition _toObjectStateTransition(
 			ListTypeEntryLocalServiceUtil.fetchListTypeEntry(
 				listTypeDefinitionId, objectStateTransition.getKey());
 
+		if ((objectStateFlowId == 0) && (sourceObjectStateId == 0)) {
+			serviceBuilderObjectStateTransition.
+				setTargetObjectStateListTypeEntryId(
+					listTypeEntry.getListTypeEntryId());
+
+			return serviceBuilderObjectStateTransition;
+		}
+
```

Comment:
@murilostodolni just out of curiosity, why do we need this?

Reply:
Hi @carolmariaabb we need this to remote `objectStateTransition` in `ObjectStateFlowLocalServiceImpl`. Without `listTypeEntryId` we won't know which transation we should go to. I talked with Feliphe Matinho about this. It's make sense?